docs/docs/code/nodejs/README.md

```diff
@@ -200,7 +206,131 @@ You can return HTTP responses from [HTTP-triggered workflows](/workflows/steps/t
 
 ## Managing state
 
-See the docs on [workflow state](/workflows/steps/code/state/).
+In Node.js (Javascript) code steps, you can also store and retrieve data in code steps.
+
+This is very useful for tracking data between runs of a particular workflow.
+
+:::warning
+This functionality (`$.service.db`) is limited to only Node.js code steps at this time.
+
+Other step languages like [Python](/code/python), [bash](/code/bash) and [Go](/code/go) do not have this feature available yet.
+
+For more information on what functionality is available for those languages, please refer to their documentation.
+:::
+
+### Injecting the database
+
+By default Node.js steps do not contain the database service. It needs to be injected by defining it as a `prop`. 
```

Comment:
Comma after "By default"

I would say, "By default, Node.js steps **don't have access to** the database service"

Reply:
👍